readme.md

```diff
@@ -1,6 +1,5 @@
40LTIuOCAyLjEtNC44IDIuMS0xLjkgMC0zLjQtLjUtNC42LTEuNi0xLjItMS4xLTEuOC0yLjQtMS44LTQuMSAwLTIgLjctMy42IDIuMi00LjYgMS41LTEuMSAzLjctMS42IDYuNS0xLjZoMi4zdi0xLjFjMC0uOS0uMi0xLjYtLjctMi4xLS40LS41LTEuMi0uOC0yLjEtLjgtLjkgMC0xLjUuMi0yIC42LS41LjQtLjcgMS0uNyAxLjdoLTUuMWMwLTEuMS4zLTIuMSAxLTMuMS43LS45IDEuNi0xLjcgMi45LTIuMiAxLjItLjUgMi42LS44IDQuMi0uOCAyLjMgMCA0LjIuNiA1LjYgMS44IDEuNCAxLjIgMi4xIDIuOCAyLjEgNXY4LjJjMCAxLjguMyAzLjIuOCA0LjF2LjJoLTUuM3ptLTQuMi0zLjVjLjggMCAxLjQtLjIgMi4xLS41LjYtLjMgMS4xLS44IDEuNC0xLjN2LTMuM2gtMS45Yy0yLjUgMC0zLjkuOS00LjEgMi42di4zYzAgLjYuMiAxLjIuNyAxLjYuNC40IDEgLjYgMS44LjZ6bTE5LjMtMi43bDMuNS0xMi44aDUuM2wtNi40IDE5aC00LjlsLTYuMy0xOWg1LjNsMy41IDEyLjh6bTE5LjYgNi42Yy0yLjggMC01LjEtLjktNi44LTIuNi0xLjgtMS43LTIuNi00LTIuNi02Ljh2LS41YzAtMS45LjQtMy42IDEuMS01LjEuNy0xLjUgMS44LTIuNyAzLjEtMy41IDEuNC0uOCAyLjktMS4yIDQuNi0xLjIgMi42IDAgNC42LjggNi4xIDIuNSAxLjUgMS42IDIuMiA0IDIuMiA3djIuMWgtMTIuMWMuMiAxLjIuNyAyLjIgMS41IDMgLjguOCAxLjkgMS4xIDMuMSAxLjEgMiAwIDMuNS0uNyA0LjYtMi4xbDIuNSAyLjhjLS44IDEuMS0xLjggMS45LTMuMSAyLjUtMS4xLjUtMi42LjgtNC4yLjh6bS0uNS0xNS43Yy0xIDAtMS44LjMtMi41IDFzLTEgMS43LTEuMiAyLjloNy4xdi0uNGMwLTEuMS0uMy0yLS45LTIuNnMtMS41LS45LTIuNS0uOXptMTYuNiAxNS4zaC01LjF2LTI3aDUuMXYyN3pNMjIwIDQxaC0xNy4xdi0xLjNsMTQuNC0yMS43aC0xMy44di0xLjZoMTYuMXYxLjNsLTE0LjUgMjEuOGgxNC45djEuNXpNMjMxLjIgNDEuNGMtMi43IDAtNC43LS44LTYuMi0yLjUtMS41LTEuNi0yLjItMy45LTIuMi02LjkgMC0yLjkuNy01LjMgMi4yLTcgMS40LTEuOCAzLjQtMi42IDUuOC0yLjYgMi4yIDAgMy45LjggNS4xIDIuMyAxLjIgMS41IDEuOSAzLjYgMS45IDYuMXYxLjNoLTEzLjJjMCAyLjUuNiA0LjQgMS44IDUuOCAxLjIgMS4zIDIuOCAyIDUgMiAxIDAgMi0uMSAyLjgtLjIuOC0uMSAxLjgtLjUgMy0uOXYxLjVjLTEgLjQtMiAuOC0yLjkuOS0xLjEuMS0yIC4yLTMuMS4yem0tLjUtMTcuNmMtMS44IDAtMy4yLjYtNC4zIDEuNy0xLjEgMS4yLTEuNyAyLjgtMS45IDVoMTEuM2MwLTIuMS0uNS0zLjgtMS40LTUtLjgtMS4xLTItMS43LTMuNy0xLjd6TTI0OS4zIDIyLjNjLjggMCAxLjYuMSAyLjUuMmwtLjMgMS42Yy0uOC0uMi0xLjYtLjMtMi40LS4zLTEuNiAwLTIuOC43LTMuOCAycy0xLjUgMy0xLjUgNXYxMC4yaC0xLjd2LTE4LjNoMS40bC4yIDMuM2guMWMuOC0xLjMgMS42LTIuMyAyLjQtMi44LjktLjYgMS45LS45IDMuMS0uOXpNMjcwIDMxLjhjMCAzLS43IDUuMy0yLjIgNy0xLjQgMS43LTMuNSAyLjUtNiAyLjUtMS42IDAtMy0uNC00LjMtMS4yLTEuMi0uOC0yLjItMS45LTIuOC0zLjMtLjctMS40LTEtMy4xLTEtNSAwLTMgLjctNS4zIDIuMi03IDEuNS0xLjcgMy40LTIuNSA2LTIuNSAyLjUgMCA0LjUuOCA1LjkgMi41IDEuNSAxLjggMi4yIDQuMSAyLjIgN3ptLTE0LjUgMGMwIDIuNS42IDQuNSAxLjcgNS45IDEuMSAxLjQgMi43IDIuMSA0LjcgMi4xczMuNi0uNyA0LjctMi4xYzEuMS0xLjQgMS43LTMuNCAxLjctNS45IDAtMi41LS42LTQuNS0xLjctNS45LTEuMS0xLjQtMi43LTIuMS00LjctMi4xcy0zLjYuNy00LjcgMi4xYy0xLjEgMS40LTEuNyAzLjQtMS43IDUuOXoiLz48bGluZWFyR3JhZGllbnQgaWQ9IlNWR0lEXzFfIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgeDE9IjI4IiB4Mj0iMjgiIHkyPSI1NiI+PHN0b3Agb2Zmc2V0PSIwIiBzdG9wLWNvbG9yPSIjNjhiZjY2Ii8+PHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjNDJiMjU3Ii8+PC9saW5lYXJHcmFkaWVudD48cGF0aCBmaWxsPSJ1cmwoI1NWR0lEXzFfKSIgZD0iTTQ4IDEzLjRsOC04di0xLjRjMC0yLjItMS44LTQtNC00aC00OGMtMi4yIDAtNCAxLjgtNCA0djM4LjZsOC04di0yNi42aDM0LjZsLTI2LjYgMjYuNi04IDgtOCA4djEuNGMwIDIuMiAxLjggNCA0IDRoNDhjMi4yIDAgNC0xLjggNC00di0zOC42bC04IDh2MjYuNmgtMzQuNmwxOS45LTE5LjkgMTQuNy0xNC43eiIvPjwvc3ZnPg=="></p>
```

Comment:
Like we discuss on twitter, you will send this same logo on PNG format in different sizes. If you want, you may commit them to the docs repo and then add the src image of the png here.

Reply:
I'm to create the pack of versions of the logo… Sorry about the delay.

Reply:
Here you have…
https://www.dropbox.com/s/edqqbdt26qscwup/laravel-zero-set.zip?dl=0

About the PR… maybe its better to close the PR for you to organize both image in the `laravel-zero/docs` repo and the README file.

D'accord? 😃